includes/apple-exporter/builders/class-components.php

```diff
@@ -140,13 +141,17 @@ private function group_body_components( $components ) {
 			$new_components[] = $body_collector;
 		}
 
-		// Trim all body components before returning
+		// Trim all body components before returning.
+		// Also set the layout for the final body component.
 		foreach ( $new_components as $i => $component ) {
 			if ( 'body' == $component['role'] ) {
 				$new_components[ $i ]['text'] = trim( $new_components[ $i ]['text'] );
+
+				if ( $i +1 === count( $new_components ) ) {
```

Comment:
no one's going to lose any sleep over it but there should be a space, `$i + 1`.  also, for this type of comparison I usually prefer to wrap with parentheses so that order of operation is explicit, even when it's not necessary